### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by Izhaki** [iamskok/use-font-face-observer#121]
Trying to use this on an NextJS app (12.1.0 with strict mode enabled), it throws a server error (for SSR):

```
Cannot use import statement outside a module
```

For this:

```
node_modules/use-font-face-observer/dist/index.modern.js
```

**Comment by iamskok** [comment]
@Izhaki what package version are you on? Can you share a repo I can check?

**Comment by iamskok** [comment]
@Izhaki Can you share a repo I can check?

**Comment by Izhaki** [comment]
https://github.com/Izhaki/use-font-face-observer/commits/master

Clone then:

```
cd repros/121
yarn dev
```

Then goto http://localhost:3000/

**Comment by iamskok** [comment]
@Izhaki thanks for providing the reproduction. The issue should be fixed in v1.2.1.
